tests/python/contrib/test_ethosu/test_type_inference.py

```diff
@@ -14,6 +14,14 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+from tvm import relay, TVMError
+from tvm import relay
+import tvm
+from tvm.relay.testing import run_opt_pass
+from .infra import make_ethosu_conv2d
+from .infra import make_ethosu_pooling
+from .infra import make_ethosu_identity
+
```

Comment:
Looks like everything got imported twice, probably when rebasing?

Reply:
Oops... removed the double imports!